pages.uk/common/tldr.md

```diff
@@ -0,0 +1,20 @@
+# tldr
+
+> Displays simple help pages for command-line tools, from the tldr-pages project.
```

Comment:
Hi @burlakvo, welcome and thanks for your submission. Where is the translation? :thinking:

Reply:
Hi @navarroaxel! I forgot save file with changes, but fixed it already